packages/in_app_purchase/README.md

```diff
@@ -202,6 +232,16 @@ InAppPurchaseConnection.instance
     .buyNonConsumable(purchaseParam: purchaseParam);
 ```
 
+### Present code redemption sheet on iOS 14.
+
+You can set up offer codes in App Store Connect for your users and let them redeem in your app. The below code will
+bring up a sheet that enables user to redeem their offer codes. For more information on redeeming offer codes, check
+out the [App Store Documentation](https://developer.apple.com/documentation/storekit/in-app_purchase/subscriptions_and_offers/implementing_offer_codes_in_your_app).
+
+```dart
+InAppPurchaseConnection.instance.presentCodeRedemptionSheet();
+```
+
 ## Development
```

Comment:
I think this section could be a part of the previous section, with a title like `After editing serialized data structs`. (What the current heading `## Development` means wasn't very clear, until I read the whole paragraph. )

Reply:
So this is actually different from the previous section. This section is more for people who contribute or develop the plugin, the previous section is more for people who uses the plugin. Maybe this can also be a part of ## Contributing to this plugin?

Reply:
SG!